src/Orleans/IDs/Interner.cs

```diff
@@ -246,10 +246,14 @@ private void InternCacheCleanupTimerCallback(object state)
                 }
             }
             long numRemoved = numEntries - internCache.Count;
-            if (numRemoved>0)
+            if (numRemoved > 0)
+            {
                 if (logger.IsVerbose) logger.Verbose(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
+            }
             else
+            {
                 if (logger.IsVerbose2) logger.Verbose2(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
```

Comment:
I think the intention is like I coded it. If something was removed, log at Verbose, if nothing was removed (less interesting event), log at Verbose2. The `if(logger.IsVerbose)` is there instead of compile time #IFDEF.